### PR TITLE
Scores/kommt und lasst uns christum ehren/reger, max

### PR DESCRIPTION
**Pull request by jonathan-scholbach**

### CODE REVIEW

File: templates/SATTB/SATTB/modern/score.ly [hunk 1]
```diff
@@ -45,14 +45,14 @@
...
-        \set Staff.instrumentName = "Tenor I"
+        \set Staff.instrumentName = "tenor I"
         \new Voice = "tenI" { \clef "G_8" \global \tenIVoice }
       >>
       \new Lyrics \lyricsto "tenI" { \tenILyrics }
 
       \new Staff \with { \consists "Ambitus_engraver" }
       <<
-        \set Staff.instrumentName = "Tenor II"
+        \set Staff.instrumentName = "tenor II"
```
Comment (fibsifan) [inline templates/SATTB/SATTB/modern/score.ly]: Why the capitalization change in the template?

File: templates/SATTB/SATTB/modern/score.ly [hunk 2]
```diff
@@ -45,14 +45,14 @@
 
       \new Staff \with { \consists "Ambitus_engraver" }
       <<
-        \set Staff.instrumentName = "Tenor I"
+        \set Staff.instrumentName = "tenor I"
```
Comment (fibsifan) [inline templates/SATTB/SATTB/modern/score.ly]: Capitalization

File: templates/SATTB/SATTB/modern/score.ly [hunk 3]
```diff
@@ -101,10 +100,9 @@
       \new Voice = "sop" { \highlightedVoice  \global \sopVoice }
       \new Voice = "alt" { \global \altVoice }
       \new Voice = "tenI" { \global \tenIVoice }
-      \new Voice = "tenII" { \global \tenIIVoice }
+      \new Voice = "tenII" { global \tenIIVoice }
```
Comment (fibsifan) [inline templates/SATTB/SATTB/modern/score.ly]: Oops

**Review by fibsifan — CHANGES_REQUESTED** [review]
You mixed changes to the template with changes to Music. I made some corrections to the music directly, but what do we do with the template?

File: templates/SATTB/SATTB/modern/score.ly (hunk 1 above)
Comment (jonathan-scholbach) [reply templates/SATTB/SATTB/modern/score.ly]: You are right, this is just a typo

File: scores/Kommt_und_lasst_uns_Christum_ehren/Kommt_und_lasst_uns_Christum_ehren_(Reger,_Max)/score.ly [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+\book {
+  \bookOutputSuffix "sopran"
+  \score {
+    \unfoldRepeats
+    \new Staff <<
+      \commonVoice
+      \new Voice = "sop" { \highlightedVoice  \global \sopVoice }
+      \new Voice = "alt" { \global \altVoice }
+      \new Voice = "tenI" { \global \tenIVoice }
+<<<<<<< HEAD
```
Comment (fibsifan) [inline scores/Kommt_und_lasst_uns_Christum_ehren/Kommt_und_lasst_uns_Christum_ehren_(Reger,_Max)/score.ly]: Leftover conflict marker.

Tipp: `git diff --check` before committing

**Review by fibsifan — CHANGES_REQUESTED** [review]
please remove the leftover conflict marker.

**Review by fibsifan — APPROVED** [review]

**Pull request merged**